Makefile

```diff
@@ -305,7 +305,7 @@ release.controller:
 	@mkdir -p $(RELEASEDIR)
 	@tar -czf $(RELEASEDIR)/$(CONTROLLER_RELEASE_NAME).tar.gz --transform 's:^.*/::' $(BINDIR)/$(CONTROLLER_APP_NAME) README.md LICENSE
 
-release.sidecar.envoy:
+release.sidecar.envoy: build.sidecar
```

Comment:
This is inconsistent with the rest of the `release.*` targets, which do not explicitly depend on the corresponding `build.*` targets. Any reason for this one to be different?